presto-mixin/dashboards/presto-worker-overview.libsonnet

```diff
@@ -0,0 +1,837 @@
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local executorPoolSizePanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'com_facebook_presto_execution_TaskManager_TaskNotificationExecutor_PoolSize{job=~"$job", presto_cluster=~"$presto_cluster", instance=~"$instance"}',
+      datasource=promDatasource,
+      legendFormat='{{presto_cluster}} - {{instance}} - task notification',
+      format='time_series',
+    ),
+    prometheus.target(
+      'com_facebook_presto_execution_executor_TaskExecutor_ProcessorExecutor_CorePoolSize{job=~"$job", presto_cluster=~"$presto_cluster", instance=~"$instance"}',
+      datasource=promDatasource,
+      legendFormat='{{presto_cluster}} - {{instance}} - process executor core',
+    ),
+    prometheus.target(
+      'com_facebook_presto_execution_executor_TaskExecutor_ProcessorExecutor_PoolSize{job=~"$job", presto_cluster=~"$presto_cluster", instance=~"$instance"}',
+      datasource=promDatasource,
+      legendFormat='{{presto_cluster}} - {{instance}} - process executor',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Executor pool size',
+  description: 'The pool size of the task notification executor, and process executor.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisBorderShow: false,
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 25,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'smooth',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'normal',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local memoryPoolPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by (instance, presto_cluster) (com_facebook_presto_memory_MemoryPool_FreeBytes{job=~"$job", presto_cluster=~"$presto_cluster", instance=~"$instance"})',
```

Comment:
do these queries need to be changed? What's the point of filtering on `instance` and `presto_cluster` if we're summing by them?

Reply:
we still need to filter based off instance and presto cluster to keep the results as those chosen by the selector no?

Reply:
yea no you're good, i think i was confusing sum and sum without when i wrote that